app/models/post.rb

```diff
@@ -1,3 +1,14 @@
 class Post < ApplicationRecord
-  belongs_to :Author_id
+  belongs_to :Author, class_name: 'User'
+  has_many :likes
+
+  after_save :update_posts_counter
+
+  def update_posts_counter
+    Author.increment!(:Posts_Counter)
+  end
+
+  def recent_comments
+    comments.limit(5).order(created_at: :desc)
```

Comment:
- [ ] Kindly note that the Post should have many comments as the association is stated in the requirements. Kinsly Add this association(`has_many :comments`)